modules/nodes-nodes-audit-log-advanced.adoc

```diff
@@ -0,0 +1,140 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-audit-log.adoc
+
+[id="nodes-nodes-audit-log-advanced-{context}"]
+= Advanced Audit  
+
+*DEPRECATED for the moment*
+
+The advanced audit feature provides several improvements over the
+basic audit functionality, including fine-grained events filtering and multiple output back ends.
+
+To enable the advanced audit feature, provide the following values in the
+`openshift_master_audit_config` parameter:
+
+----
+openshift_master_audit_config={"enabled": true, "auditFilePath": "/var/lib/origin/oscp-audit.log", "maximumFileRetentionDays": 14, "maximumFileSizeMegabytes": 500, "maximumRetainedFiles": 5, "policyFile": "/etc/origin/master/adv-audit.yaml", "logFormat":"json"}
+----
+
```

Comment:
Where to set the `openshift_master_audit_config` parameter ? How to set it ?

Reply:
@zhouying7780 I apologize for the confusion. The advanced audit feature that this file describes is not in 4.1.